libr/core/config.c

```diff
@@ -224,6 +219,21 @@ static int cb_asmarch(void *user, void *data) {
 	//if (!strcmp (node->value, "bf"))
 	//	r_config_set (core->config, "dbg.backend", "bf");
 	__setsegoff (core->config, node->value, core->assembler->bits);
+	
+	// set a default endianness
+	int isbig;
+	int isbigbin = r_bin_is_big_endian(core->bin);
```

Comment:
missing space. isbigbin sounds like bool, but expects -1? the isbig one should be bool